src/libraries/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -248,6 +248,8 @@ namespace System.Drawing
 
         public static Color Purple => new Color(KnownColor.Purple);
 
+        public static Color RebeccaPurple => new Color(KnownColor.RebeccaPurple);
```

Comment:
Thanks.  You will need to update the ref assembly as well by adding an entry in https://github.com/dotnet/runtime/blob/9162830bc651fbff0ab1957c0155d4ecf375109e/src/libraries/System.Drawing.Primitives/ref/System.Drawing.Primitives.cs

Reply:
ok i will do this soon.